src/core/fonts.js

```diff
@@ -5326,15 +5326,21 @@ var Type1Parser = (function Type1ParserClosure() {
   }
 
   function decrypt(data, key, discardNumber) {
-    var r = key | 0, c1 = 52845, c2 = 22719;
-    var count = data.length;
+    if (discardNumber > data.length) {
```

Comment:
Should this be `>=`? If `disardNumber` equals `data.length`, we also discard all entries so we can also return an empty array.